src/workers/sequencingDiagnostics.ts

```diff
@@ -162,3 +167,183 @@ function generateAbsoluteTimeStringDiagnostics(
 
   return diagnostics;
 }
+
+/**
+ * Find a list of all command nodes (at the level of C.COMMAND_NAME)
+ *                                                     ^^^^^^^^^^^^
+ * @param sourceFile The root node to start looking under
+ * @returns The list of command nodes, all the way back at the time elements
+ */
+function findCommandNodes(sourceFile: tsc.Node): tsc.Node[] {
```

Comment:
This is a great solution for handling the commands specified in the `step: []` section of the sequence. We have commands in the `immediate_commands` and `requests` You were probably where unaware of these parts of the `sequence`

ex.

```ts
immediate_commands : [PREHEAT_OVEN({temperature : 10}],
request : [C.REQUEST('string', C.COMMAND_NAME, C.COMMAND_NAME)), REQUEST('string', C.COMMNAND_NAME, A`2024-004T00:00:000`.COMMAND_NAME]
```

C.REQUEST is trying to be validated which causes false positive errors in the editor. I